package.json

```diff
@@ -10,8 +10,8 @@
     "android-test": "./run android-tests",
     "ios-test": "./run ios-tests",
     "clean": "rm -rf node_modules",
-    "lint": "eslint '**/*.js'",
-    "lint-fix": "eslint '**/*.js' --fix"
+    "lint": "eslint \"**/*.js\"",
+    "lint-fix": "eslint \"**/*.js\" --fix"
```

Comment:
Was it intentional to replace single quotes with escaped double quotes?

Reply:
Yes. was attempting to fix my linter, and I don't think it has any impact across platforms. If it does I can change

Reply:
I was just curious if it was some automated tool or an actual issue. Thanks :)